src/Microsoft.SqlTools.ServiceLayer/ExecutionPlan/ExecutionGraph/Comparison/SkeletonManager.cs

```diff
@@ -6,7 +6,7 @@
 using System.Collections.Generic;
 using System.Linq;
 
-namespace Microsoft.SqlTools.ServiceLayer.ShowPlan.ShowPlanGraph.Comparison
+namespace Microsoft.SqlTools.ServiceLayer.ExecutionPlan.ExecutionGraph.Comparison
 {
     /// <summary>
     /// Handles operations for creating and comparing skeletons of showplan trees
```

Comment:
Looks like you didn't get everything, make sure you're doing a search over the entire solution to catch them all

Reply:
I didn't rename any names that included "ShowPlan" in files under the "ExecutionGraph" directory since @aasimkhan30 said it would it difficult to port over code from SSMS in the future.

Reply:
But this isn't specific to showplan, right? So it doesn't seem like it should reference that directly. 

@aasimkhan30 ?